src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -1,31 +1,51 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import path from 'ramda/src/path';
 import pathOr from 'ramda/src/pathOr';
+import styled from 'styled-components';
 import { Link } from 'react-router-dom';
-import { GhostGrid, GridItemConstrainedMedium } from '#lib/styledGrid';
+import { GhostGrid, GridItemConstrainedLarge } from '#lib/styledGrid';
+import { RequestContext } from '#contexts/RequestContext';
+
 import MetadataContainer from '../Metadata';
 import LinkedData from '../LinkedData';
 import headings from '../Headings';
 import timestamp from '../ArticleTimestamp';
 import text from '../Text';
 import image from '../Image';
 import Blocks from '../Blocks';
+import MediaPlayer from '../MediaPlayer';
 import ATIAnalytics from '../ATIAnalytics';
 import cpsAssetPagePropTypes from '../../models/propTypes/cpsAssetPage';
 
-const componentsToRender = {
-  headline: headings,
-  text,
-  image,
-  timestamp,
-};
-
 const CpsAssetPageMain = ({ pageData }) => {
+  const { platform } = useContext(RequestContext);
   const title = path(['promo', 'headlines', 'headline'], pageData);
   const summary = path(['promo', 'summary'], pageData);
   const metadata = path(['metadata'], pageData);
+  const assetUri = path(['locators', 'assetUri'], metadata);
   const blocks = pathOr([], ['content', 'model', 'blocks'], pageData);
 
+  const componentsToRender = {
+    headline: headings,
+    text,
+    image,
+    timestamp,
+    video: props => (
+      <MediaPlayer
```

Comment:
This should be in its own file

Reply:
This function?  why?

Reply:
Its a container in its own right, we shouldnt just shove code into other containers, it should be isolated and unit tested. This is the pattern across simorgh :)

Reply:
👍 will move it, although fairly sure moving such a simple helper component to a new file will make this less pleasant to work on

Reply:
The original is still here..